app/models/participating_seller.rb

```diff
@@ -0,0 +1,15 @@
+# == Schema Information
+#
+# Table name: participating_sellers
+#
+#  id             :bigint           not null, primary key
+#  name           :string
+#  stamp_url      :string
+#  tickets_secret :string
+#  created_at     :datetime         not null
+#  updated_at     :datetime         not null
+#  seller_id      :bigint
+#
+class ParticipatingSeller < ApplicationRecord
+    validates_presence_of :name, :tickets_secret
```

Comment:
tickets_secret you don't need to validate the presence of. Matter of fact we want for it to be nullable